### PR TITLE
feature: added lists

### PR DESCRIPTION
**Pull request by omgaikwad**
feature: added lists
- unordered list styles
- ordered list styles

### CODE REVIEW

File: components/lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}
+.list-style-smAlpha {
+  list-style-type: lower-alpha;
+}
+.list-style-smRoman {
+  list-style-type: lower-roman;
+}
+.list-style-capRoman {
+  list-style-type: upper-roman;
+}
```
Comment (hinanshisuthar) [inline components/lists/lists.css:35]: add a new line at the EOF

**Review by hinanshisuthar — COMMENTED** [review]
looks good to me.

File: components/lists/lists.css (hunk 1 above)
Comment (omgaikwad) [reply components/lists/lists.css:35]: Sure

**Review by FarhanMobashir — COMMENTED** [review]
✅✅

**Pull request merged**